src/foundation.cs

```diff
@@ -12879,7 +12892,8 @@ interface NSTextCheckingResult : NSSecureCoding, NSCopying {
 		double TimeInterval { get; }
 
 		[Export ("components")]
-		[Availability (Introduced = Platform.iOS_4_0 | Platform.Mac_10_7)]
+		[Availability (Introduced = Platform.iOS_4_0)]
```

Comment:
Same question about availability.  IIRC we don't support all the way back to iOS 4.0 anyway, do we need to keep the tags around?

Reply:
yes, we can drop < 6.0